smurfs/src/actions/index.js

```diff
@@ -1,15 +1,62 @@
-/* 
-  Action Types Go Here!
-  Be sure to export each action type so you can pull it into your reducer
-*/
-
-/*
-  For this project you'll need at least 2 action creators for the main portion,
-   and 2 more for the stretch problem.
-   Be sure to include action types for each type of action creator. Also, be sure to mind
-     the "pending" states like, fetching, creating, updating and deleting.
-   C - addSmurf
-   R - getSmurfs
-   U - updateSmurf
-   D - deleteSmurf
-*/
+import axios from 'axios';
+
+export const FETCH_START = 'FETCH_START';
+export const FETCH_SUCCESS = 'FETCH_SUCCESS';
+export const FETCH_FAILURE = 'FETCH_FAILURE';
+export const ADD_START = 'ADD_START';
+export const SET_UPDATE = 'SET_UPDATE';
+export const UPDATE_START = 'UPDATE_START';
```

Comment:
You're on the right track. It would be a great idea to also have an `ADD_FAIL`, `ADD_SUCCESS`, and such.